velox/functions/sparksql/JsonObjectKeys.h

```diff
@@ -39,6 +39,12 @@ struct JsonObjectKeysFunction {
       return false;
     }
 
+    // The result is NULL if the given string is not a valid JSON string.
+    // such as "invalid json"
```

Comment:
nit: perhaps remove this line as it is covered by test.

Reply:
sure, remove it